package.json

```diff
@@ -82,14 +88,15 @@
     "react-dom": "^17.0.1",
     "react-redux": "^7.2.1",
     "react-router-dom": "^5.2.0",
-    "redux": "^4.0.0"
-  },
-  "dependencies": {
+    "react-test-renderer": "17.0.1",
+    "redux": "4.0.5",
     "@fortawesome/free-solid-svg-icons": "^5.8.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
+    "algoliasearch": "^4.8.5",
     "classnames": "^2.2.5",
     "query-string": "^5.1.1",
     "react-instantsearch-dom": "^6.8.3",
     "react-loading-skeleton": "^2.1.1"
-  }
+  },
+  "dependencies": {}
```

Comment:
is an empty list needed?

Reply:
I'm not sure it's needed, but I feel like, in the absence of being able to have comments in JSON, explicitly leaving it in lets everyone know that this was a deliberate choice.